pkg/core/tokens/default_signing_key.go

```diff
@@ -48,18 +48,18 @@ func (d *defaultSigningKeyComponent) Start(stop <-chan struct{}) error {
 func (d *defaultSigningKeyComponent) createDefaultSigningKeyIfNotExist() error {
 	_, _, err := d.signingKeyManager.GetLatestSigningKey()
 	if err == nil {
-		log.V(1).Info("user token's signing key already exists. Skip creating.")
+		d.log.V(1).Info("signing key already exists. Skip creating.")
 		return nil
 	}
 	if _, ok := err.(*SigningKeyNotFound); !ok {
 		return err
 	}
-	log.Info("trying to create user token's signing key")
+	d.log.Info("trying to create signing key")
 	if err := d.signingKeyManager.CreateDefaultSigningKey(); err != nil {
-		log.V(1).Info("could not create user token's signing key", "err", err)
+		d.log.V(1).Info("could not create signing key", "err", err)
```

Comment:
It's intentional. We may not be able to create a token because webhook fails or mesh is not ready. We will inform about errors after all retries but we don't want to scare users with errors that comes from eventually consistency.

Reply:
I'm taking this as optional since you approved a PR. If you feel strongly about this change, please let me know so we can work on that.